### ISSUE TITLE
ParallelStream is not available

### ISSUE THREAD

**Issue by luk3thomas** [beatrichartz/csv#56]
Hi!

I recently created release with edeliver and I'm seeing this?

```
** (exit) an exception was raised:^M
    ** (UndefinedFunctionError) function ParallelStream.map/3 is undefined (module ParallelStream is not available)^M
        ParallelStream.map(#Function<57.89908360/2 in Stream.transform/3>, #Function<1.67996581/1 in CSV.Decoder.decode/2>, [num_workers: 3])^M
```

Naive question, but would requiring `ParallelStream` in the `Decoder` module fix the runtime error I am seeing?

**Comment by beatrichartz** [comment]
Probably you want to load parallel stream as part of your applications, however that should be done by this library in the current version - which version are you using?

**Comment by luk3thomas** [comment]
From mix.lock

```
|    "csv": {:hex, :csv, "1.4.3", "de92e2bc85bc09ec676f19f6ec2daa59ba1f0103d1da96cf0cca36342c90d93f", [:mix], [{:parallel_stream, "~> 1.0.4", [hex: :parallel_stream, optional: false]}]},
```

Yeah, I thought about that. I experimented loading parallel stream in my own application as an application dependency. The application wouldn't start since parallel stream isn't a direct dependency.

**Comment by beatrichartz** [comment]
What does your application definition in mix.exs look like, does it load csv? CSV itself loads parallel_stream as an application dependency, so it should be loaded as part of that.

**Comment by luk3thomas** [comment]
Here is an except of my mix file:

```elixir
  def application do
    [mod: {Aprit, []},
     applications: [:phoenix, :phoenix_pubsub, :phoenix_html, :cowboy, :logger, :gettext,
                    :phoenix_ecto, :postgrex, :guardian, :csv, :edeliver]]
  end

  defp deps do
    [{:phoenix, "~> 1.2.1"},
     {:phoenix_pubsub, "~> 1.0"},
     {:phoenix_ecto, "~> 3.0"},
     {:postgrex, ">= 0.0.0"},
     {:phoenix_html, "~> 2.6"},
     {:phoenix_live_reload, "~> 1.0", only: :dev},
     {:guardian, "~> 0.12.0"},
     {:gettext, "~> 0.11"},
     {:poison, "~> 1.0"},
     {:csv, "~> 1.4.2"},
     {:cowboy, "~> 1.0"},
     {:edeliver, "1.3.0"},
     {:distillery, ">= 0.8.0", warn_missing: false}]
  end
```

Oh I see that it does require parallel stream as an application dependency. Hmmm, weird. I'm probably doing something wrong. If you don't mind, I'll go ahead close the issue for now and I'll reopen it if I find anything else.

**Comment by davidkuhta** [comment]
@luk3thomas Did you find a solution? I have this same issue and have already tried adding `:parallel_stream` to my `applications:` with no luck.

**Comment by beatrichartz** [comment]
Might this be related to this issue https://github.com/boldpoker/edeliver/issues/157 ?

**Comment by davidkuhta** [comment]
@beatrichartz Thanks, but I was able to build my releases. 
After much trial and error, I did a fresh build and it worked.
Could've just been the other RTFM (with R=Restart & M=Machine)  ¯\_(ツ)_/¯

**Comment by luk3thomas** [comment]
I think https://github.com/beatrichartz/csv/pull/51 should fix the problem I am seeing. I suppose the fix will be included in an upcoming release sometime soon?

**Comment by beatrichartz** [comment]
@luk3thomas I released this in 1.4.4 today, sorry for the wait
